spec/factories/shopping_list_items.rb

```diff
@@ -2,9 +2,9 @@
 
 FactoryBot.define do
   factory :shopping_list_item do
-    shopping_list
+    association :list, factory: :shopping_list
 
-    description { 'Necklace' }
```

Comment:
Had to change this after adding the validation that description had to be unique for items on a given list.